src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -82,6 +82,10 @@ function WorkspaceInitialPage(props) {
         Navigation.navigate(ROUTES.SETTINGS_WORKSPACES);
     }, [props.reports, policy]);
 
+    useEffect(() => {
+        Policy.unsetLoadingFlag();
+    }, [])
```

Comment:
Why do we need this if the flag is already unset when creating an workspace succeeds?